GL-796: Adds dynamic applicable exemptions form

alexdesi opened the pull request:
### Jira link

https://transformuk.atlassian.net/browse/GL-796

### What?

- [x] Adds a form object model to validate dynamic applicable category assessment exemptions and tie in with a govuk formbuilder ui
- [x] Adds a bunch of separate controllers for each of the wizard steps
- [x] Removes use of session globally to support multiple tabs 

### Why?

- This enables a user to submit answers to different category assessment exemptions for categories 1 and 2 as part of the green lanes wizard

shjohnson reviewed (dismissed):
Want to get this merged so can interact with other pages so approving and will fix forward in other PR's

willfish committed:
GL-796: Adds dynamic applicable exemptions form

This supports our ability to validate incoming answers
to category 1 and 2 exemptions and helps to
plumb together the overall journey

Co-authored-by: Alexdesi <alexdesi@gmail.com>
